storage/backup/backup-cli/src/storage/mod.rs

```diff
@@ -190,3 +190,37 @@ impl StorageOpt {
         })
     }
 }
+
+#[derive(Parser)]
+#[clap(group(
+    ArgGroup::new("storage")
+    .required(true)
+    .args(&["local-fs-dir", "command-adapter-config"]),
+))]
+pub struct DBToolStorageOpt {
+    #[clap(
+        long,
+        conflicts_with = "command-adapter-config",
```

Comment:
I don't think we need group and conflicits_with at the same time. Just set the group here?

Reply:
yes, removed the conflicts_with